packages/api-browser/rollup.config.js

```diff
@@ -1,3 +1,24 @@
+// import resolve from 'rollup-plugin-node-resolve';
+// import commonjs from 'rollup-plugin-commonjs';
+
+// export default {
+//   entry: 'bundle.js',
+//   moduleName: 'ssf',
+//   format: 'umd',
+//   plugins: [
+//     resolve({
+//       module: true,
+//       main: true,
+//       browser: false,
+//       jsnext: true
+//     }),
+//     commonjs({
+//       include: '**'
+//     })
+//   ],
+//   dest: 'build/dist/containerjs-api.js'
+// };
+
 import resolve from 'rollup-plugin-node-resolve';
```

Comment:
Yep, that's what I meant. It's `rollup-plugin-node-resolve` that does this isn't it?

`api-browser` already has that, but `api-bundle` didn't (so I had to add it when I created `api-utility`). You've added it to `api-openfin` already, so I think it's all good 👍